src/lib/y2partitioner/dialogs/commit.rb

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "yast/i18n"
+require "cwm/dialog"
+require "y2partitioner/widgets/commit_actions"
+
+Yast.import "Label"
+
+module Y2Partitioner
+  module Dialogs
+    # Dialog to show the progress of the storage changes while they are being applied
+    class Commit < CWM::Dialog
+      include Yast::I18n
+
+      def initialize
+        textdomain "storage"
+      end
+
+      def title
+        _("Performing Installation")
```

Comment:
As far as I understood, this new dialog is only shown in a running system. Isn't it? If so, should we choose a different title?

Reply:
Yes, it is only used in a running system. I have simply used the same title as the old partitioner, but I agree, it seems this is not the most accurate name in the world. What about "Applying Changes To The System".

I am not sure about capitalizing "To The", what do you think?

Reply:
"Applying Changes to the System" looks good to me. 

https://english.stackexchange.com/questions/14/which-words-in-a-title-should-be-capitalized